File: curso-python/ex034.py
# escreva um programa que:

#  pergunte o salário de um funcionário
salario = float(input("digite o salário do funcionário: "))

#  calcule o valor do seu aumento:

#para salários superiores a R$1.250,00, calcule um aumento de 10%
if salario > 1250.00:
    aumento = salario * (10/100)
    novo_salario = salario + aumento
    print(f"o salário do funcionário é de {salario} e com aumento de 10% ficou {novo_salario}")


# para os inferiores ou iguais, o aumento é de 15%
if salario <= 1250:
    aumento = salario * (15/100)
    novo_salario = salario + aumento
    print(f"o salário funcionário é de {salario} e com aumento de 15% ficou {novo_salario}")

File: curso-python/javascript-exercicios/ex034.js
const prompt = require("prompt-sync")();

// Pergunta o salário
const salario = parseFloat(prompt("Digite o salário do funcionário: "));
let aumento = 0;
let novoSalario = 0;

// Verifica se o salário é superior a 1250
if (salario > 1250) {
    aumento = salario * 0.10;
    novoSalario = salario + aumento;
    console.log(`O salário do funcionário é de R$${salario.toFixed(2)} e com aumento de 10% ficou R$${novoSalario.toFixed(2)}`);
} else {
    aumento = salario * 0.15;
    novoSalario = salario + aumento;
    console.log(`O salário do funcionário é de R$${salario.toFixed(2)} e com aumento de 15% ficou R$${novoSalario.toFixed(2)}`);
}
